### PR TITLE
Fix build issues (on OpenSUSE 15.1) in Release30 branch

### PR DESCRIPTION
**Pull request by thiller2018**
"make rpms" didn't work.

Also added SLES12 SP5 to distros

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -88,10 +88,10 @@ release: clean qa readme authors sdist doc
 	@mkdir release
 	@cp dist/*.gz release/
 	@cp cobbler.spec release/
-	@cp debian/cobbler.dsc release/
```
Comment (SchoolGuy) [inline Makefile]: Would you mind removing them. We removed the debian packaging folder completly as we are doing a cross platform build in the OBS currently.

File: Makefile [hunk 2]
```diff
@@ -187,7 +187,7 @@ restartservices:
 rpms: release
 	mkdir -p rpm-build
 	cp dist/*.gz rpm-build/
-	cp patches/*.patch rpm-build/
```
Comment (SchoolGuy) [inline Makefile]: Same as above ^

**Comment by thiller2018** [comment]
Change done

**Comment by SchoolGuy** [comment]
@thiller2018 Would you mind opening this against master also?

**Pull request merged**